### PR TITLE
Fixed tests suite

### PR DESCRIPTION
**Pull request by clash82**
Nothing more to say :)

### CODE REVIEW

**Comment by clash82** [comment]
review ping @bdunogier

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
     "name": "ezsystems/marketing-automation-bundle",
     "description": "eZ Platform Marketing Automation",
+    "require":
+    {
+        "ezsystems/ezpublish-kernel": "^5.0 | ^6.0"
```
Comment (clash82) [inline composer.json]: I'm not sure about this part?

Comment (bdunogier) [inline composer.json]: Does it work on 5.3 and earlier ? if yes, it's okay.

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+    },
     "authors": [
         {
             "name": "eZ Systems",
```
Comment (bdunogier) [inline composer.json]: eZ web team ? ;)

**Comment by bdunogier** [comment]
+1

File: composer.json (hunk 2 above)
Comment (andrerom) [inline composer.json]: :P
We have eZ Systems everywhere afaik, or at least we should.

**Comment by andrerom** [comment]
+1, but seems to have conflicts, also we should setup travis so you see it passes as well.

**Comment by clash82** [comment]
Cannot rebase this. New PR was submitted https://github.com/ezsystems/MarketingAutomationBundle/pull/10

**Pull request closed without merging**